client/src/app/EmptyTab.js

```diff
@@ -47,21 +49,28 @@ export default class EmptyTab extends PureComponent {
     );
   }
 
-  renderCloudColumn = () => {
+  renderPlatform8Column = () => {
```

Comment:
We decided to only change `Platform`, `Cloud` ->  `Platform 7`, `Platform 8` in labels and texts visible to the user, not technical stuff. So let's not change the function names as it would contradict the rest of the codebase.